### DISCUSSION TITLE
Stack vs. native-stack

### DISCUSSION THREAD

**Discussion by devoren** [expo/router#887] [Q&A]
Hello! I use expo-router in my app. Since I use native stack, I have a problem with an external library. Is there a difference between native stack and js stack in expo-router? What are the disadvantages of using a stack compared to a native stack?

**Comment by marklawlor** [comment]
Expo Router uses React Navigation's Stack and Native Stack. You can refer to their documentation which explains the differences between the two

- https://reactnavigation.org/docs/stack-navigator
- https://reactnavigation.org/docs/native-stack-navigator

**Reply by devoren** [reply]
<img width="199" alt="image" src="https://github.com/expo/router/assets/99968085/cf0aa02d-58f3-4b00-bffc-914645cce43d">

I mean expo-router uses its native stack, so if I change it to stack, will any features not work on stack?
